lib/grpc/status.ex

```diff
@@ -0,0 +1,113 @@
+defmodule GRPC.Status do
+  
+  @doc """
+  Not an error; returned on success.
+  """
+  def ok, do: "0"
```

Comment:
Why no use int instead of string? When sending that on the wire, we can change the int to string.

Reply:
I had it as an int at first but if I return int here we can't match against the values returned from the servers without converting them again as they are returned as strings.

Reply:
But enum seems a natural choice. Anyway, we can keep them strings and can change them later only if other will not use the literal values.